SpherePacking/ForMathlib/PoissonSummation/DualLattice.lean

```diff
@@ -31,7 +31,7 @@ section Preliminaries
 end Preliminaries
 
 /-- The dual of a `ZLattice` is its dual as a ℤ-submodule. -/
-def ZLattice.Dual : Submodule ℤ E := LinearMap.BilinForm.dualSubmodule B Λ
+noncomputable def ZLattice.Dual : Submodule ℤ E := LinearMap.BilinForm.dualSubmodule B Λ
```

Comment:
This is a weird change, I'm guessing it's related to the new compiler. Would you mind trying to make a MWE for why this thing is noncomputable now? I'd expect it to be erased since a submodule is a set (ie prop-valued data) and a bunch of proofs, all of which should disappear at compile time.

Reply:
This is the error: 

> failed to compile definition, consider marking it as 'noncomputable' because it depends on 'Real.normedField', which is 'noncomputable'